### ISSUE TITLE
Release tarball is named `SDL_sound`, not `SDL2_sound`

### ISSUE THREAD

**Issue by sezero** [icculus/SDL_sound#36]
@icculus: The release tarball extracts as `SDL_sound-2.0.1` and not as
`SDL2_sound-2.0.1`: is this an oversight?

**Comment by icculus** [comment]
The "SDL2_sound" name creeped into some parts of the source, like the CMake file, and I think we named the binary SDL2_sound.so to match other SDL libraries, but the project is still called SDL_sound.

**Comment by sezero** [comment]
It doesn't match the other SDL2 projects, though.

**Comment by icculus** [comment]
Oh, I thought we changed the others.

We should definitely fix that and call it 2.0.2 right away.  :/

**Comment by sezero** [comment]
> We should definitely fix that and call it 2.0.2 right away. :/

Yep. Although, I don't know how to do the trick with github's release machinery.
Another job for you :/

**Comment by icculus** [comment]
Okay, wait, I've definitely got "libSDL2_ttf", "libSDL2_image", and "libSDL2_mixer" libraries in my /usr/local/lib, so this is at least the intended name on Unix systems.

**Comment by icculus** [comment]
Oh, you mean the tarball filename doesn't match other SDL_* libraries. Hmm, let me see how to change that on GitHub's release thing.

**Comment by icculus** [comment]
I don't think I can rename it; this works on SDL_mixer and such because we're just hosting the tarballs at libsdl.org and can name them whatever we want (the GitHub version there is "SDL_mixer-release-2.0.4.tar.gz" !!! ), but GitHub just puts the repository name in front of the tag name here.

**Comment by sezero** [comment]
Maybe we can specifically upload a proper source tarball with the releases like some other projects do - see [harfbuzz](https://github.com/harfbuzz/harfbuzz) or [libsamplerate](https://github.com/libsndfile/libsamplerate) for example.

**Comment by madebr** [comment]
You can create a source release with git using:
```
git archive v2.0.1 -o SDL2_sound-2.0.1.tar.gz --prefix "SDL2_sound-2.0.1/"
```
sign that archive with your gpg key, and add it to https://github.com/icculus/SDL_sound/releases/tag/v2.0.1

**Comment by thesamesam** [comment]
Yeah, it'd be great if you could consider doing that for future, as they're guaranteed to return stable results, unlike github commit-based tarballs.

**Comment by ericoporto** [comment]
Hey, if a new release was made with the changes in main and with the mentioned fix here, it would be really great.

**Comment by icculus** [comment]
I think I did this right, with a manually archived tarball and GPG signature: https://github.com/icculus/SDL_sound/releases/tag/v2.0.2

Please let me know if I screwed this up and I'll try again.  :)
